tests/rpc/rpc_05-state_test.go

```diff
@@ -34,10 +36,17 @@ func TestStateRPCResponseValidation(t *testing.T) { //nolint:tparallel
 	t.Run("state_call", func(t *testing.T) {
 		t.Parallel()
 
-		const params = `["", "","0x580d77a9136035a0bc3c3cd86286172f7f81291164c5914266073a30466fba21"]`
-		var response modules.StateCallResponse
+		const params = `["Core_version", "0x"]`
+		var response runtime.Version
 
-		fetchWithTimeout(ctx, t, "state_call", params, &response)
+		rpcCtx, rpcCancel := context.WithTimeout(ctx, 10*time.Second)
+		defer rpcCancel()
+		endpoint := rpc.NewEndpoint(node.RPCPort())
+		data, err := rpc.Post(rpcCtx, endpoint, "state_call", params)
+		require.NoError(t, err)
+
+		err = rpc.DecodeScale(data, &response)
+		require.NoError(t, err)
 
 		// TODO assert stateCallResponse
```

Comment:
what about this todo?

Reply:
https://github.com/ChainSafe/gossamer/issues/3229

Reply:
Could you add this issue number to the comment please

Reply:
done